packages/frontend-api/src/Resources/config/services.yml

```diff
@@ -5,7 +5,11 @@ services:
         public: false
 
     Shopsys\FrontendApiBundle\:
-        resource: '../../**/*{Resolver,Map}.php'
+        resource: '../../**/*{Facade,Map,Resolver}.php'
+
+    Shopsys\FrontendApiBundle\Model\Mutation\:
+        resource: '../../Model/Mutation/'
+        tags: ['overblog_graphql.mutation']
```

Comment:
Why is it necessary to tag mutation services, when Resolvers are not and without the tag, it works.
(I assume you used this https://github.com/overblog/GraphQLBundle/blob/master/docs/definitions/resolver.md#the-service-way)

Reply:
159be85a16709a2efae20f1e598baaeb2614b0e3

Reply:
What about adding Mutation into the general definition on line 8?
like 
```diff
    Shopsys\FrontendApiBundle\:
-       resource: '../../**/*{Facade,Map,Resolver}.php'
+       resource: '../../**/*{Facade,Map,Mutation,Resolver,}.php'
```

instead of a separate definition?

Reply:
41dfa325e9f286aeef4b6f70ca940962ac311db7